src/bloom.cpp

```diff
@@ -59,17 +59,15 @@ void CBloomFilter::insert(const COutPoint& outpoint)
 {
     CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
     stream << outpoint;
-    std::vector<unsigned char> data(stream.begin(), stream.end());
-    insert(data);
+    insert(Span<const unsigned char>((const unsigned char*)stream.data(), stream.size()));
```

Comment:
Could simply use the recently introduced `MakeUCharSpan()` helper here and on other occurences of the commit.